src/Hazelcast.Net/DistributedObjects/ReliableTopicMessageExecutor.cs

```diff
@@ -125,7 +120,7 @@ private async Task ExecuteAsync(CancellationToken cancellationToken)
 
                 if (lostCount != 0 && !_options.IsLossTolerant)
                 {
-                    _logger.IfWarning()?.LogWarning("The reliable topic subscription [{Id}] is not loss tolerant. {Count} message(s) is lost. Disposing the process...", _id, (-1 * lostCount));
+                    _logger.IfWarning()?.LogWarning("The reliable topic subscription [{Id}] is not loss tolerant. {Count} message(s) is lost. Terminating the process...", _id, (-1 * lostCount));
```

Comment:
I would write `Terminating the subscription`. And then I assume we'll get the `Terminated` event? the "process" feels like we're going to shutdown the application.

Reply:
Makes sense, fixed at https://github.com/hazelcast/hazelcast-csharp-client/pull/825/commits/29bfc61d64d9fcf6984c77fe142aac9fa8d43173